rns_shell/platform/graphics/libwpe/WindowLibWPE.cpp

```diff
@@ -147,6 +149,8 @@ Window* Window::createNativeWindow(void* platformData) {
         delete window;
         return nullptr;
     }
+    if(!mainWindow_)
```

Comment:
As a TODO Action: does it Need to check the created window type for Main Window

Reply:
Ideally first window will be Main Window,  So if  it is `NULL`, it means there is no Main Window yet.. And we set it to `mainWindow` to `NULL` when we close/destroy the Main Window which would only happen when app exits.